docs/confilctClient.md

```diff
@@ -0,0 +1,151 @@
+---
```

Comment:
typo in file name. Kinda drawback that we get to have entire files as new. Would it be possible to to `git reset --soft...` and do not rename files?